tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -1,9 +1,37 @@
 import logging
 import unittest
 
-from backend.corpora.common.corpora_orm import ProjectStatus, DbDataset, DbUser
+from backend.corpora.common.corpora_orm import (
+    ProjectLinkType,
+    DbProjectLink,
+    ProcessingState,
+    ValidationState,
+    ProjectStatus,
+    DbDataset,
+    DbUser,
+)
 from backend.corpora.common.entities.entity import logger as entity_logger
 from backend.corpora.common.entities.project import Project
+from tests.unit.backend.corpora.common.entities.utils import get_ids
+
+
+class ProjectParams:
+    count = 0
```

Comment:
What is the purpose of `count`? There isn't really a need for the project name to be unique...

Reply:
it will be easier to identify during debug. It can also be extended to generate more complicated test inputs in the future.

Reply:
the uuids take longer to compare when look at them at a glance.

Reply:
Sorry, perhaps I still don't understand. Doesn't each test have it's own error message? So you never need to know what project UUID is associated with each test. 

What are the more complicated test inputs that you envision? We can always bring this back when the complicated test cases you envision actually materialize instead of adding it now.

Reply:
I have already found it useful in debugging the tests. We can remove it later if it stops being useful.

Reply:
It helped me when testing within a time range that only projects within that time range were retrieved.

Reply:
Also I don't want to spend time generating unique names to help my self identify projects. I think as  we start to add more input validation have a function or class to generate the inputs will be very helpful.